PhysicsTools/PatAlgos/python/slimming/lostTracks_cfi.py

```diff
@@ -10,14 +11,16 @@
     primaryVertices = cms.InputTag("offlineSlimmedPrimaryVertices"),
     originalVertices = cms.InputTag("offlinePrimaryVertices"),
     muons = cms.InputTag("muons"),
-    minPt = cms.double(0.95),	
-    minHits = cms.uint32(8),	
-    minPixelHits = cms.uint32(1),  
-    covarianceVersion = cms.int32(0), #so far: 0 is Phase0, 1 is Phase1   
-    covarianceSchema = cms.int32(0), #old miniaod like
+    minPt = cms.double(0.95),
+    minHits = cms.uint32(8),
+    minPixelHits = cms.uint32(1),
+    covarianceVersion = cms.int32(0), #so far: 0 is Phase0, 1 is Phase1
+    #covariancePackingSchemas = cms.vint32(1,257,513,769,0),  # a cheaper schema in kb/ev
+    covariancePackingSchemas = cms.vint32(8,264,520,776,0),  # more accurate schema
```

Comment:
Unless you plan to have eventually sone different schema, to avoid desynchronizations, you could better import it from PhysicsTools/PatAlgos/python/slimming/packedPFCandidates_cfi.py

Reply:
Yes, will be done